### PR TITLE
Add pricing and start page

### LINKED ISSUE #118: Create UI for SKALE chain refilling
SKALE Portal should have a page regarding to SKALE chain payment. It should show current SKL token price, current basic chain price in USD and SKL. You can specify a chain and you can see a counter when your lifetime is ended(1 day of unpayed month)

Add timer on Network UI which shoes how many HH:MM:SS left for next payment
 - if it is more than 5 days - it would be green
 - If less than 5 days - it would be orange
 - if payment missed - it would be red and shown a message with a amount of months which schain was not payed and info message what to do in this situation
 
 See PRD:
 https://www.notion.so/skalelabs/PRD-SKALE-Chain-Pricing-8fecc72884c54b2987ba05d7ef89fdaa

### LINKED ISSUE #143: Add dedicated apps page and start page
We need to enhance the user experience of the SKALE Portal by introducing two new pages: 'Apps' page and a 'Start' page. These additions are aimed at streamlining navigation and improving discoverability of features and applications within the SKALE ecosystem.

1. Apps Page
Objective: Create a new page titled 'Apps' which will serve as a discovery platform for users to find and interact with a curated list of applications hosted on the SKALE network. Should be available under `/apps` url.

2.  Start Page
Objective: Add a 'Start' page that will serve as the default landing page when users access the SKALE Portal. This page should offer a dashboard-like experience with cards linking to all the main portal pages (Bridge, Chains, Apps, Stats, etc.).

The problem: currently, the default page is Bridge, which forces users to accept Terms and Conditions for the Bridge before they can proceed to other pages.

Screenshot 2023-11-09 at 15 35 51

### PR DESCRIPTION
**Pull request by dmytrotkk**
This PR adds new pages and functionality to the SKALE Portal. 

New Home page will now become the default entrypoint to the Portal website, Terms of service modal now showed only for Bridge-related pages. Resolves #143.

This PR also adds sChain pricing functionality, which resolves #118.

### CODE REVIEW

**Comment by TheGreatAxios** [comment]
General Notes
1. Cannot get it to load see image below. Not sure if broken? or maybe due to network I'm on? 
2. Without actively looking for it, cannot find the manage tab for a chain 

image

File: src/components/ChainCard.tsx [hunk 1]
```diff
@@ -27,22 +27,17 @@ import {
...
-import UnfoldMoreRoundedIcon from '@mui/icons-material/UnfoldMoreRounded'
-
 import ChainLogo from './ChainLogo'
-import { getExplorerUrl } from '../core/chain'
 
 import { MAINNET_CHAIN_LOGOS } from '../core/constants'
 
 export default function ChainCard(props: { skaleNetwork: interfaces.SkaleNetwork; schain: any[] }) {
-  const explorerUrl = getExplorerUrl(BASE_EXPLORER_URLS[props.skaleNetwork], props.schain[0])
+  // const explorerUrl = getExplorerUrl(BASE_EXPLORER_URLS[props.skaleNetwork], props.schain[0])
```
Comment (TheGreatAxios) [inline src/components/ChainCard.tsx:40]: Remove comment if unused

File: src/components/HelpZen.tsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
+import { Link } from 'react-router-dom'
+
+import React, { useEffect, useState, MouseEvent } from 'react'
```
Comment (TheGreatAxios) [inline src/components/HelpZen.tsx:3]: General Note -> You are importing React and useState and then using React.useState below.
Unless compiler requires React to be imported at the top level suggest using the functions directly for better bundle size

File: src/components/MonthSelector.tsx [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+import { useState } from 'react'
+import Button from '@mui/material/Button'
+import TextField from '@mui/material/TextField'
+
+import { cmn, cls } from '@skalenetwork/metaport'
+
+import { formatTimePeriod } from '../core/timeHelper'
+
+const _MONTH_RECOMMENDATIONS = [1, 2, 3, 6, 12, 18, 24]
```
Comment (TheGreatAxios) [inline src/components/MonthSelector.tsx:32]: For style purposes - suggest changing to MONTH_RECOMMENDATIONS

File: src/pages/Start.tsx [hunk 4]
```diff
@@ -0,0 +1,68 @@
+import Container from '@mui/material/Container'
+import Stack from '@mui/material/Stack'
+import Box from '@mui/material/Box'
+import Grid from '@mui/material/Grid'
+
+import SwapHorizontalCircleOutlinedIcon from '@mui/icons-material/SwapHorizontalCircleOutlined'
+import PublicOutlinedIcon from '@mui/icons-material/PublicOutlined'
+import InsertChartOutlinedIcon from '@mui/icons-material/InsertChartOutlined'
+// import AppsOutlinedIcon from '@mui/icons-material/AppsOutlined'
+// import WalletOutlinedIcon from '@mui/icons-material/WalletOutlined'
```
Comment (TheGreatAxios) [inline src/pages/Start.tsx:10]: Remove unused icons if not needed

**Review by TheGreatAxios — APPROVED** [review]
Approved - Left some general comments and notes for cleanup.

File: src/components/MonthSelector.tsx (hunk 3 above)
Comment (dmytrotkk) [reply src/components/MonthSelector.tsx:32]: changed

File: src/components/HelpZen.tsx (hunk 2 above)
Comment (dmytrotkk) [reply src/components/HelpZen.tsx:3]: updated

File: src/components/ChainCard.tsx (hunk 1 above)
Comment (dmytrotkk) [reply src/components/ChainCard.tsx:40]: removed

**Pull request merged**